pkg/cmd/pulumi/new.go

```diff
@@ -423,28 +451,16 @@ func newNewCmd() *cobra.Command {
 		// Show default help.
 		defaultHelp(cmd, args)
 
-		// Attempt to retrieve available templates.
-		repo, err := workspace.RetrieveTemplates("", false /*offline*/, workspace.TemplateKindPulumiProject)
-		if err != nil {
-			logging.Warningf("could not retrieve templates: %v", err)
-			return
-		}
-
-		// Get the list of templates.
-		templates, err := repo.Templates()
+		templates, err := getTemplates()
 		if err != nil {
 			logging.Warningf("could not list templates: %v", err)
 			return
 		}
 
 		// If we have any templates, show them.
 		if len(templates) > 0 {
-			available, _ := templatesToOptionArrayAndMap(templates, true)
-			fmt.Println("")
-			fmt.Println("Available Templates:")
-			for _, t := range available {
-				fmt.Printf("  %s\n", t)
-			}
+			fmt.Println()
+			fmt.Printf("You have %d templates available. To view them interactively, run `pulumi new`.\n", len(templates))
```

Comment:
This is the only place in the command where we address the user. 
```suggestion
			fmt.Printf("There are %d templates available. To view them interactively, run `pulumi new`.\n", len(templates))
```

Reply:
I could probably alter this text more since I addded the --list-templates command